torch/csrc/jit/python/script_init.cpp

```diff
@@ -1219,8 +1220,10 @@ void initJitScriptBindings(PyObject* module) {
         auto typed_inputs = toTraceableStack(input_tuple);
         std::shared_ptr<Graph> graph = std::get<0>(tracer::createGraphByTracing(
             func, typed_inputs, var_lookup_fn, strict, force_outplace));
+
         auto cu = get_python_cu();
         auto name = c10::QualifiedName(qualname);
+        NormalizeOps(graph);
```

Comment:
Is this also needed in `_create_method_from_trace`? Can we just embed this call in `trace()` (called by `createGraphByTracing`)?  Maybe here:
https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/frontend/tracer.cpp#L454